espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/DrawerActionHandler.kt

```diff
@@ -28,8 +28,8 @@ class DrawerActionHandler(private val isOpenAction: Boolean) : RequestHandler<Dr
     override fun handle(params: DrawerActionParams): Void? {
         val viewInteraction = Element.getViewInteractionById(params.elementId)
         try {
-            params.gravity?.let {
-                viewInteraction.perform(if (isOpenAction) DrawerActions.open(it) else DrawerActions.close(it))
+            params.gravity?.let {gravity ->
```

Comment:
I just discovered that you can explicitly name the variable instead of `it`.... I think this way is better because it's more explicit. What do you think @mykola-mokhnach ?

Reply:
whenever is more convenient for you. IDEA anyway highlights all the occurrences of `it`